.vscode/launch.json

```diff
@@ -6,7 +6,18 @@
     "configurations": [
         {
             "name": "Frontend",
-            "command": "pnpm start",
+            "command": "pnpm start-http",
```

Comment:
Aren't both scripts doing the same `start-http` now?

Reply:
Lol, yes, let me fix it

Reply:
Done